docs/integrations/webhooks/sync-data.mdx

```diff
@@ -1,77 +1,86 @@
 ---
-title: Sync Clerk data to your application with webhooks
-description: Learn how to sync Clerk data to your application with webhooks.
+title: Sync Clerk data to your app with webhooks
+description: Learn how to sync Clerk data to your app with webhooks.
 ---
 
 <TutorialHero
   beforeYouStart={[
-    {
-      title: "Set up a Clerk + Next.js application",
+
+      {
+      title: "A Clerk + Next.js app is required.",
       link: "/docs/quickstarts/nextjs",
       icon: "nextjs",
+    },
+    {
+      title: "A ngrok account is required.",
+      link: "https://dashboard.ngrok.com/signup",
+      icon: "user-circle",
     }
   ]}
 >
-  - Listen for an event in the Clerk Dashboard
-  - Create a webhook handler that:
-    - uses Svix to verify the webhook signature
-    - receives the webhook's payload
-  - Test the webhook locally using ngrok and the Clerk Dashboard
-  - Add logic to your application to trigger the webhook
+  - Set up ngrok
+  - Set up a webhook endpoint
+  - Create the webhook
+  - Get type inference for your webhook events
+  - Test the webhook
+  - Configure your production instance
 </TutorialHero>
 
-The recommended way to sync data between Clerk and your application is to use webhooks.
+The recommended way to sync Clerk data to your app is through webhooks.
+
+In this guide, you'll set up a webhook in your Next.js app to listen for the `user.created` event, create an endpoint in the Clerk Dashboard, build a handler, and test it locally using ngrok and the Clerk Dashboard.
 
-In this guide, you will learn how to create a Clerk webhook in your Next.js application. You will listen for the `user.updated` event by creating a webhook endpoint in the Clerk Dashboard, creating a webhook handler in your Next.js application, and testing the webhook locally using ngrok and the Clerk Dashboard.
+Clerk offers many user events, but three key events include:
```

Comment:
```suggestion
Clerk offers many events, but three key events include:
```
`user.created`, `user.updated` and `user.deleted` are the only user events. The others are for things like Organizations, Organization Memberships, Organization Invitations, Sessions, etc